test/acceptance/test/add_tests.go

```diff
@@ -279,18 +279,18 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 		})
 	})
 
-	It("Verify 'wego app add' with --dry-run flag does not modify the cluster", func() {
+	It("Verify wego app add . --dry-run flag does not modify the cluster", func() {
 		var repoAbsolutePath string
 		var session *gexec.Session
 		private := true
-		branchName := "test-branch"
+		branchName := "main"
 		appManifestFilePath := "./data/nginx.yaml"
 		defaultSshKeyPath := os.Getenv("HOME") + "/.ssh/id_rsa"
 		appRepoName := "wego-test-app-" + RandString(8)
 		workloadName := "nginx"
 		workloadNamespace := "my-nginx"
-		url := "ssh://git@github.com/" + os.Getenv("GITHUB_ORG") + "/" + appRepoName + ".git"
-		addCommand := "app add . --url=" + url + " --branch=" + branchName + " --dry-run"
+		url := "ssh://git@github.com/weaveworks-gitops-test/" + appRepoName + ".git"
```

Comment:
@rokshana-b Would it be ok to use the env var instead?. It looks like you are reverting a previous change right? Something happened?

Reply:
I just saw the description of this PR 🙂

Reply:
My tests passed with `SmokeTests` tag, but the acceptance suite failed as a whole when I removed the tags. Something went very wrong... I'm reverting all changes